fili-core/src/main/java/com/yahoo/bard/webservice/web/CsvResponseWriter.java

```diff
@@ -0,0 +1,56 @@
+// Copyright 2017 Oath Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.web;
+
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.fasterxml.jackson.dataformat.csv.CsvSchema;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Collections;
+
+/**
+ * CSV writer for response.
+ */
+public class CsvResponseWriter implements ResponseWriter {
+    /**
+     * Writes CSV response.
+     *
+     * @param request  ApiRequest object with all the associated info in it
+     * @param responseData  data object containing all the result information
+     * @param os  OutputStream
+     *
+     * @throws IOException if a problem is encountered writing to the OutputStreamC
+     */
+    public void write(
```

Comment:
Should be annotated with [`@Override`](https://stackoverflow.com/questions/94361/when-do-you-use-javas-override-annotation-and-why)